dev/com.ibm.ws.webcontainer.servlet.3.1_fat/fat/src/com/ibm/ws/webcontainer/servlet31/fat/tests/WCServerTest.java

```diff
@@ -235,7 +238,7 @@ public void testServlet31() throws Exception {
      *                       if something goes horribly wrong
      */
     @Test
-    @SkipForRepeat(SkipForRepeat.NO_MODIFICATION)
+    @SkipForRepeat({NO_MODIFICATION, EE9_FEATURES})
```

Comment:
I'm not quite sure what the point of this test is vs the servlet31 test above. Are these even worth keeping around?